internal/pkg/bulk/opMulti_test.go

```diff
@@ -10,13 +10,14 @@ import (
 	"testing"
 
 	testlog "github.com/elastic/fleet-server/v7/internal/pkg/testing/log"
+	"github.com/rs/zerolog/log"
 )
 
 const payload = `{"_id" : "1", "_index" : "test"}`
 
 // Test throughput of creating multiOps
 func BenchmarkMultiUpdateMock(b *testing.B) {
-	_ = testlog.SetLogger(b)
+	log.Logger = testlog.SetLogger(b)
```

Comment:
I've had to make a few changes like this; the `testlog.SetLogger` method creates a zerolog logger that will write to testing output (so integration tests look cleaner) and had a side effect of setting `log.Logger` to it. However the side effect was causing race issues in tests